pkg/src/index.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as exec from '@actions/exec';
+import * as toolCache from '@actions/tool-cache';
+import * as os from 'os';
+import * as tmp from 'tmp';
+import { promises as fs } from 'fs';
+import { getReleaseURL } from './format-url';
+import * as downloadUtil from './download-util';
+import * as installUtil from './install-util';
+import { getLatestGcloudSDKVersion } from './version-util';
+
+export { getLatestGcloudSDKVersion };
+
+/**
+ * @Method: Check if gcloud is installed
+ * @Param {string} version (Optional) Cloud SDK version
+ * @Return {string} tool path for Cloud SDK
+ */
+export function isInstalled(version?: string) {
+  let toolPath;
+  if (version) {
+    toolPath = toolCache.find('gcloud', version);
+  } else {
+    toolPath = toolCache.findAllVersions('gcloud');
```

Comment:
Note, this could return an array if there are multiple gcloud versions installed.